src/components/head-meta/index.tsx

```diff
@@ -0,0 +1,26 @@
+import Head from "next/head";
+
+type HeadMetaProps = {
+  title: string;
+  description?: string;
+  keywords?: string;
+  favicon?: string;
+};
```

Comment:
Could you kindly move these types into a separate file that is specifically for types, please?